FrxEditor.inc

```diff
@@ -421,17 +421,26 @@ class FrxEditor {
       'default');
   }
 
-
-  public function addParameters($parms_to_add) {
+  /**
+   * Add parameters to report
+   * @param array $parms_to_add, boolean $keyValue
+  */
+  public function addParameters($parms_to_add, $keyValue = false) {    
     $parms_to_add = (array)$parms_to_add;
-    foreach ($parms_to_add as $parm) {
-      $parms= array();
-      $parms[$parm] = array('id' => $parm );
-      $this->setFrxHeader('parameters', 'parm',
+    $parms= array();
+    if ($keyValue) {        
+      foreach ($parms_to_add as $key=>$parm) {        
+        $parms[$key] = array('id' => $parm['id'], 'label' => $parm['label']);    
```

Comment:
I would still prefer to see this assignment be more flexible, per this comment:
https://github.com/ulsdevteam/forena/pull/7#commitcomment-25515648